src/types/redis_json.h

```diff
@@ -66,6 +66,7 @@ class Json : public Database {
 
   std::vector<rocksdb::Status> MGet(const std::vector<std::string> &user_keys, const std::string &path,
                                     std::vector<JsonValue> &results);
+  rocksdb::Status MSet(const std::vector<std::string> &user_keys, const std::vector<std::string> &paths, const std::vector<std::string> &values);
```

Comment:
Aligned with the behavior of redis, no sub set is executed when an error occurs.